README.md

```diff
@@ -1,3 +1,7 @@
+[![Conda package](https://github.com/IntelPython/mkl_umath/actions/workflows/conda-package.yml/badge.svg)](https://github.com/IntelPython/mkl_umath/actions/workflows/conda-package.yml)
+
+[![Scorecard supply-chain security](https://github.com/IntelPython/mkl_umath/actions/workflows/openssf-scorecard.yml/badge.svg)](https://github.com/IntelPython/mkl_umath/actions/workflows/openssf-scorecard.yml)
```

Comment:
think it would be better to put them visually in to the same line

```suggestion
[![Conda package](https://github.com/IntelPython/mkl_umath/actions/workflows/conda-package.yml/badge.svg)](https://github.com/IntelPython/mkl_umath/actions/workflows/conda-package.yml)
[![Scorecard supply-chain security](https://github.com/IntelPython/mkl_umath/actions/workflows/openssf-scorecard.yml/badge.svg)](https://github.com/IntelPython/mkl_umath/actions/workflows/openssf-scorecard.yml)
```